packages/ui/src/components/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -18,74 +18,71 @@ import {
   Thead,
   Tr,
 } from "@patternfly/react-table";
-import type { FunctionComponent } from "react";
+import { useCallback, useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import type { Consumer } from "ui-models/src/models/consumer-group";
 import "../ConsumerGroup.css";
 import type { OffsetValue } from "../types";
 import { OffsetSelect } from "./OffsetSelect";
 import { TopicSelect } from "./TopicSelect";
+import type { ConsumerGroupState } from "@rhoas/app-services-ui-components";
```

Comment:
Don't import from ui-components, all the Kafka related stuff from there will be removed eventually. You should have this as `ConsumerGroup['state']` with `ConsumerGroup` imported from `ui-models` (I think).